src/test/java/org/jnode/fs/ntfs/TestNtfsReparsePoint.java

```diff
@@ -0,0 +1,37 @@
+package org.jnode.fs.ntfs;
+
+import org.jnode.fs.ntfs.attribute.ReparsePointAttribute;
+import org.jnode.fs.ntfs.attribute.NTFSAttribute;
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+import static org.jnode.fs.FileSystemTestUtils.*;
+
+/**
+ * Tests for reparse point data in {@link NTFSAttribute}.
+ */
+public class TestNtfsReparsePoint {
+
+    @Test
+    public void testResidentReparsePoint()  {
+        byte[] data = intsToByteArray(
+                "-64 0 0 0 -56 1 0 0 0 0 0 0 0 0 3 0 -84 1 0 0 24 0 0 0 3 0 0 -96 -92 1 0 0 0 0 -48 0 -46 0 " +
```

Comment:
Maybe we can save this as binary test data?

Reply:
I learned it from the other test classes in the same package, they all just put a byte array like this.